profiles/customhooks.py

```diff
@@ -59,8 +57,9 @@ def IVField() -> str:
             return iv_field
 
         def PhaseSummary() -> dict:
+            from modules.stats import total_stats  # TODO prevent instantiating TotalStats class before profile selected
```

Comment:
Moved import here to prevent a circular import - might just pass the encounter rate as a var to the custom hooks func?